### DISCUSSION TITLE
Open Planner: scheduling and managing conference talks

### DISCUSSION THREAD

**Discussion by bpetetot** [conference-hall/conference-hall#311] [Tools & tips]
**Open Planner** is a SaaS app for scheduling and managing conference talks. Add/edit speakers and sessions, manage tracks/rooms, and handle sponsors.

It integrates with Conference Hall to schedule accepted talks.

- **Application:** https://openplanner.fr
- **Repository:** https://github.com/HugoGresse/OpenPlanner
- **Support it:** https://github.com/sponsors/HugoGresse

**Comment by GerardPaligot** [comment]
I noticed that it’s now possible to schedule directly in Conference Hall. Can you clarify the positioning between Conference Hall’s internal scheduling and its integration with OpenPlanner?

Currently, it seems we can’t export the schedule from Conference Hall. Would it make sense to finalize the initial schedule in Conference Hall first and then export it to OpenPlanner once it’s ready? The process isn’t entirely clear from now.

**Reply by bpetetot** [reply]
For now, Schedule management in Conference Hall and OpenPlanner are completely separate.

**OpenPlanner integration** allows you to export/update Conference Hall proposals in OpenPlanner. (available with an export action in the proposal list).

**Schedule management in Conference Hall** is still in beta and is a very limited feature. We're still thinking about what kind of exports or APIs we should provide, and about its possible integration with Open Planner. For the moment, nothing has been decided or ruled out on this subject.

**Reply by HugoGresse** [reply]
to clarify on OpenPlanner part, OpenPlanner act as a midleware between services. 
While ConferenceHall aims to be an internal service for organizer, OpenPlanner act as a backend and possible website for conferences. Meaning OpenPlanner can have public oriented frontend, like the schedule and FAQ, tickets prices, sponsors, etc but can also be used with other CFP services. 
Based on my last discussion with Benjamin, this seems to make the orientation more clear for ConferenceHall and OpenPlanner, one tool that does everything, possibly public, another only for the schedule and CFP, internal.  Let me know if I'm correct or not Benjamin.
